lib/components/atoms/Input/Input.js

```diff
@@ -20,20 +20,21 @@ const Input = React.forwardRef<Props, HTMLInputElement>(
   (
     { id, className, type, name, value, placeholder, ariaLabel, elementLocator, ...others }: Props,
     ref
-  ): Node => (
-    <input
-      {...others}
-      id={id}
-      ref={ref}
-      aria-label={ariaLabel}
-      className={className}
-      name={name}
-      type={type}
-      value={type !== 'file' ? value : undefined}
-      placeholder={placeholder}
-      data-locator={elementLocator || (type ? `input-${type}-${name}` : '')}
-    />
-  )
+  ) => {
+    const propArray = {
+      id,
+      ref,
+      'aria-label': ariaLabel,
+      className,
+      name,
+      type,
+      placeholder,
+      'data-locator': elementLocator || `input-${type}-${name}`,
+      ...others,
```

Comment:
@ankush29 Please look at Achal how he has fixed flow errors. I think one of the main issue here i see it about uses of ...others prop

I think as per flow this should be first prop and specific ones should be set later. @achaljain can help please connect